### PR TITLE
Set up documentation on read the docs

### PR DESCRIPTION
**Pull request by hmpf**
## Scope and purpose

Secures the zino namespace on read the docs.

## Contributor Checklist

Every pull request should have this checklist filled out, no matter how small it is.
More information about contributing to Zino can be found in the
[README](https://github.com/Uninett/zino/blob/master/README.md#developing-zino).

* [X] Added a changelog fragment for [towncrier](https://github.com/Uninett/zino/blob/master/README.md#before-merging-a-pull-request)
* [ ] Added/amended tests for new/changed code 

There are no code changes, hence no tests.

* [X] Added/changed documentation
* [ ] Linted/formatted the code with black, ruff and isort, easiest by using [pre-commit](https://github.com/Uninett/zino/blob/master/README.md#code-style)

There are no code changes, hence no linting.

* [X] The first line of the commit message continues the sentence "If applied, this commit will ...", starts with a capital letter, does not end with punctuation and is 50 characters or less long. See https://cbea.ms/git-commit/
* [ ] If applicable: Created new issues if this PR does not fix the issue completely/there is further work to be done

### CODE REVIEW

**Comment by hmpf** [comment]
Black complains about "docs/conf.py" which is autogenerated.

File: changelog.d/+add-docs-dir.added.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added directory for sphinx docs in order to setup up read the docs and secure
+the name "zino" there.
```
Comment (johannaengland) [inline changelog.d/+add-docs-dir.added.md:2]:
```suggestion
Added directory for sphinx docs in order to set up read the docs and secure
the name "zino" there.
```

**Review by johannaengland — APPROVED** [review]

File: .readthedocs.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```
Comment (lunkwill42) [inline .readthedocs.yaml:12]: Zino itself will not currently run on Python 3.12 (some of its dependencies won't import).  Although the docs are currently empty, if we wish to auto-import some of the modules during doc generation, we should do so on a Python version that Zino supports (i.e. 3.11, for now).

```suggestion
    python: "3.11"
```

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Zino'
+copyright = '2024, Morten Brekkevold, Johanna England, Simon Tveit'
```
Comment (lunkwill42) [inline docs/conf.py:10]: We cannot claim personal copyright here :-)

Either

```suggestion
copyright = '2024, Sikt'
```

or 


```suggestion
copyright = '2024, Sikt - The Norwegian Agency for Shared Services in Education and Research'
```

**Review by lunkwill42 — CHANGES_REQUESTED** [review]
Looks decent, just nitpicking details :)

**Review by lunkwill42 — APPROVED** [review]

**Pull request merged**